### PR TITLE
[Ubuntu] Add pass package

### LINKED ISSUE #2302: Add pass credential helper
**Tool information**
- Tool name: pass
- Tool license: GPLv2
- Add
- Desired version: Nothing specific. The one from standard apt repo is ok.
- Approximate size:
- If this is an add request:
  - Brief description of tool: The standard unix password manager
  - URL for tool's homepage: https://www.passwordstore.org/
  - Provide a basic test case to validate the tool's functionality:

**Area for Triage**:

**Feature**:

We need a standard [credential store](https://docs.docker.com/engine/reference/commandline/login/#credentials-store) like [`pass` credential helper](https://www.passwordstore.org/) on Ubuntu GitHub Runners to handle Docker login in a safest way (see docker/login-action#30). Otherwise credentials are stored unencrypted:

```
🔑 Logging into docker.pkg.github.com...
WARNING! Your password will be stored unencrypted in /home/runner/.docker/config.json.
Configure a credential helper to remove this warning. See
https://docs.docker.com/engine/reference/commandline/login/#credentials-store
```

**Virtual environments affected**
- [x] Ubuntu 16.04
- [x] Ubuntu 18.04
- [x] Ubuntu 20.04
- [ ] macOS 10.15
- [ ] macOS 11.0
- [ ] Windows Server 2016 R2
- [ ] Windows Server 2019

**Can this tool be installed during the build?**

**Tool installation time in runtime**

**Are you willing to submit a PR?**

### PR DESCRIPTION
**Pull request by crazy-max**
# Description

Adds `pass` package

#### Related issue:

Fixes #2302 

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

cc. @maxim-lobanov

### CODE REVIEW

**Comment by crazy-max** [comment]
@maxim-lobanov Can you fire up an Azure pipeline just to make sure the regex is ok :sweat_smile:

```
$ pass version
============================================
= pass: the standard unix password manager =
=                                          =
=                  v1.7.3                  =
=                                          =
=             Jason A. Donenfeld           =
=               Jason@zx2c4.com            =
=                                          =
=      http://www.passwordstore.org/       =
============================================
```

File: images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1 [hunk 1]
```diff
@@ -175,6 +175,11 @@ function Get-PackerVersion {
     return "Packer $(packer --version)"
 }
 
+function Get-PassVersion {
+    $passVersion = pass version | Select-String "^=\s+v").Line.Replace('v','') | Take-OutputPart -Part 2
```
Comment (maxim-lobanov) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1:179]: I think you need one more "(" before `pass version`
Suggest this way to simplify `(pass version | Select-String "v([\d\.]+)").Matches.Value.Replace('v', '')`

**Comment by maxim-lobanov** [comment]
In general, looks good. I have left minor comment.

File: images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1 (hunk 1 above)
Comment (crazy-max) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1]: Done

Comment (crazy-max) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1]: Maybe another linter for powershell scripts would be nice here :)

**Comment by crazy-max** [comment]
@maxim-lobanov Argh looks like the [regex is wrong](https://github.visualstudio.com/virtual-environments/_build/results?buildId=93047&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=72559e73-928c-58e0-b67e-e22d7a78faae&l=82).

**Comment by crazy-max** [comment]
@maxim-lobanov 

* [16.04](https://github.visualstudio.com/virtual-environments/_build/results?buildId=93084&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=72559e73-928c-58e0-b67e-e22d7a78faae&l=80): pass 1.6.5
* [18.04](https://github.visualstudio.com/virtual-environments/_build/results?buildId=93085&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=72559e73-928c-58e0-b67e-e22d7a78faae&l=82): pass 1.7.1
* [20.04](https://github.visualstudio.com/virtual-environments/_build/results?buildId=93086&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=72559e73-928c-58e0-b67e-e22d7a78faae&l=82): pass 1.7.3

**Review by maxim-lobanov — APPROVED** [review]

**Comment by maxim-lobanov** [comment]
> @maxim-lobanov
> 
> * [16.04](https://github.visualstudio.com/virtual-environments/_build/results?buildId=93084&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=72559e73-928c-58e0-b67e-e22d7a78faae&l=80): pass 1.6.5
> * [18.04](https://github.visualstudio.com/virtual-environments/_build/results?buildId=93085&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=72559e73-928c-58e0-b67e-e22d7a78faae&l=82): pass 1.7.1
> * [20.04](https://github.visualstudio.com/virtual-environments/_build/results?buildId=93086&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=72559e73-928c-58e0-b67e-e22d7a78faae&l=82): pass 1.7.3

Great, 
@actions/virtual-environments-akvelon could you please review and approve it too?

**Review by kadirselcuk — APPROVED** [review]

**Review by Darleev — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

**Review by MaksimZhukov — APPROVED** [review]

**Pull request merged**